tests/testsuite/cargo/help/mod.rs

```diff
@@ -0,0 +1,12 @@
+use cargo_test_support::curr_dir;
```

Comment:
Just curious, is there a reason `curr_dir` is not in `cargo_test_support::prelude`?

Reply:
Not really. I can add it if wanted.

Reply:
That might help with some of the boilerplate. It looks like it is used for essentially every UI test. This is unrelated to this PR, though, and can be done any time.